easybuild/easyconfigs/b/BAGEL/BAGEL-1.2.2.patch

```diff
@@ -0,0 +1,88 @@
+diff -ruN bagel-1.2.2/src/molecule/shell_base.cc bagel-1.2.2.orig/src/molecule/shell_base.cc
```

Comment:
@TopRichard could you add a comment here explaining the purpose of the patch please?

Reply:
The patch is to resolve missing boost includes and fix ambiguous function call, similar to https://github.com/qsimulate-open/bagel/pull/229

Reply:
@TopRichard please move the comment to the top of the patch (above the diff line) and also add the patch author.
also, don't forget to update the checksum of the patch.

Reply:
Done